src/appleseed-max-impl/appleseedenvmap/appleseedenvmap.h

```diff
@@ -0,0 +1,152 @@
+#pragma once
+
+// appleseed.foundation headers.
+#include "foundation/platform/windows.h"    // include before 3ds Max headers
+#include "foundation/utility/autoreleaseptr.h"
+#include "renderer/modeling/environmentedf/environmentedf.h"
+
+// 3ds Max headers.
+#include "appleseedenvmap/resource.h"
```

Comment:
Add a separate section for appleseed-max header files:

    // appleseed-max headers.
    #include "appleseedenvmap/resource.h"